static/alert.ts

```diff
@@ -98,7 +98,8 @@ export class Alert {
         autoDismiss = true,
         dismissTime = 5000,
     }: AlertNotifyOptions) {
-        const container = $('#notifications');
+        // JQuery<HTMLElement> | null is the correct type, @types/jquery has the wrong typing here
```

Comment:
Are you sure about this one? https://api.jquery.com/jQuery/#jQuery-element mentions:
> If no elements match the provided selector, the new jQuery object is "empty"; that is, it contains no elements and has .length property of 0.

Reply:
Oh oops my bad, thank you for catching that! I made a mistake while testing locally. I presume the next line is supposed to be a check along the lines of `if (container.length == 0) return;`. Would it be good to get some sort of error logging for if something like this happens though?